src/main/java/frc/robot/controllers/NeoDrivetrain.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.controllers;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import frc.robot.util.*;
+
+public class NeoDrivetrain extends Drivetrain {
+    public CANSparkMax leftMotor1, leftMotor2, rightMotor1, rightMotor2;
+    private static PID leftDrivePID = new PID(0.09, 0, 0);
+    private static PID rightDrivePID = new PID(0.09, 0, 0);
```

Comment:
Wouldn't all instances of NeoDrivetrain share the the same instance of leftDrivePID, since there is only one instance per class due to "static"? I wonder, maybe just initializing those in constructor the same way as the motors would be sufficient.